src/GraphQL/Execution/ErrorInfoProvider.cs

```diff
@@ -53,8 +54,11 @@ public virtual ErrorInfo GetInfo(ExecutionError executionError)
                     codes = null;
                 var number = _options.ExposeCode && executionError is ValidationError validationError ? validationError.Number : null;
                 var data = _options.ExposeData && executionError.Data?.Count > 0 ? executionError.Data : null;
+                var stacktrace = _options.ExposeExceptionStackTrace && _options.ExposeExceptionStackTraceMode == ExposeExceptionStackTraceMode.Extensions
+                    ? BuildStackTrace(executionError)
```

Comment:
```suggestion
                    ? executionError.ToString()
```
I would just store `ToString()` which contains all the details, including the type name and description of each inner exception.

Example:

```
System.NullReferenceException: Outer exception message
 ---> System.InvalidOperationException: Center exception message
 ---> System.ApplicationException: Innermost exception message
   at GraphQL.MicrosoftDI.Tests.AdapterTests.AllowNullProvider() in C:\Users\Shane\source\repos\graphql-dotnet\graphql-dotnet\src\GraphQL.MicrosoftDI.Tests\AdapterTests.cs:line 26
   --- End of inner exception stack trace ---
   at GraphQL.MicrosoftDI.Tests.AdapterTests.AllowNullProvider() in C:\Users\Shane\source\repos\graphql-dotnet\graphql-dotnet\src\GraphQL.MicrosoftDI.Tests\AdapterTests.cs:line 30
   --- End of inner exception stack trace ---
   at GraphQL.MicrosoftDI.Tests.AdapterTests.AllowNullProvider() in C:\Users\Shane\source\repos\graphql-dotnet\graphql-dotnet\src\GraphQL.MicrosoftDI.Tests\AdapterTests.cs:line 35
```

Perhaps the property name should be called "details" instead of "stackTrace", however.